modules/persistent-storage-local-uninstall-operator.adoc

```diff
@@ -7,12 +7,24 @@
 
 To uninstall the Local Storage Operator, you must remove the Operator and all created resources in the `local-storage` project.
 
+[WARNING]
+====
+It is not recommended to uninstall the Local Storage Operator while local storage PVs are still in use. While the PVs will remain after the Operator's removal,
```

Comment:
Uninstalling the Local Storage Operator while local storage PVs are still in use is not recommended.